README.md

```diff
@@ -63,64 +89,103 @@ Untracked files:
 
-Git allows us to mark changes to our code as different versions, or, in Git speak, "commits." A commit is like a frozen copy of your code at a given point. Once you've made a commit, you can always easily revert to that version of your code as it existed at that exact moment.
+Commits are amazingly powerful in `git`. With a commit you can move your project
+back to a past commit and throw away bad ideas. You can tell git to create a new
+"parallel universe" based off of a past commit and you can start working in that
+universe and decide later to keep that work or throw it away. We won't cover
+these features in this lesson, but they exist, and are very useful.
 
-Now that Git is aware of a change to our project –– the addition of `README.md` –– let's submit the first official version of our project using `git commit`. Whenever we make a commit, we must supply a commit message describing the version or commit. This message log makes it easy for us to figure out what each commit or version is all about.
+Now that Git is aware of a change to our project –– the addition of `README.md`
+–– let's submit the first official version of our project using `git commit`.
+Whenever we make a commit, we must supply a commit message describing the version
+or commit. This message log makes it easy for us to figure out what each commit
+or version is all about.
 
-To make your first commit, type: `git commit -m "Added README.md"`. This tells Git that our commit message, represented by the `-m` flag, is `"Added README.md"`.
+To make our first commit, type: `git commit -m "Initial commit"`. This tells
+Git that our commit message, represented by the `-m` flag, is `"Initial commit"`.
 
 ```
-next-big-thing $ git commit -m "Added README.md"
-[master (root-commit) e55477d] Added README.md
+my-git-project $ git commit -m "Initial commit"
+[master (root-commit) e55477d] Initial commit
  1 file changed, 0 insertions(+), 0 deletions(-)
  create mode 100644 README.md
 ```
 
-Git tells us that it created a new version of our code, represented by the esoteric SHA `e55477d` (basically how Git keeps track of versions). The commit changed 1 file. With that commit made and no other changes to our files, if we ask Git what the status of our project is now, we'll see that it is at a "Clean State", that there is nothing to commit and no new changes.
+We can see that `git` has created a new version of our code, represented by the
+_SHA_ `e55477d` (the identification system that Git uses to keep track
+of versions). The commit changed 1 file. Now, if we type `git status` ask to
+see the status of our project, with no other changes to any files, we'll see
+that it is at a "clean state", and there is nothing to commit and no new changes.
 
 ```
-next-big-thing $ git status
+my-git-project $ git status
 On branch master
 nothing to commit, working directory clean
 ```
+If we make another change, for example, to README.md, we can follow the same steps
```

Comment:
This is inaccurate b/c you would have to do `-a` (unless it were already added). Can we broaden this out and have them make an edit to README.md and then use the `-am` chant described below